database/dynamodb.go

```diff
@@ -29,34 +21,50 @@ type AWSCredentials struct {
 var db *dynamodb.DynamoDB
 
 func init() {
+	defer func() {
+		if err := recover(); err != nil {
+			fmt.Println(err)
+		}
+	}()
 
-	env, found := os.LookupEnv(Constants["ENV"])
-	if !found {
-		// load the env values using the `setUpEnv` function.
-		utils.SetUpEnv()
+	env := os.Getenv(utils.ENV)
+	if env == utils.PROD {
+		log.Println(env, " is the env")
 	}
 
 }
 
 func getAWSCredentials() *AWSCredentials {
 
+	var found bool
 	awsCredentials := new(AWSCredentials)
-	awsCredentials.Region = os.Getenv(Constants["REGION"])
-	awsCredentials.AccessKey = os.Getenv(Constants["ACCES_KEY"])
-	awsCredentials.SecretKey = os.Getenv(Constants["SECRET_KEY"])
+
+	awsCredentials.Region, found = os.LookupEnv(utils.REGION)
```

Comment:
The os.LookupEnv function is used to check if environment variables are present, but it doesn't assign their values to the awsCredentials struct. The correct function to assign the environment variables' values is os.Getenv. Therefore, the os.LookupEnv calls should be replaced with os.Getenv calls.

Reply:
rather than getenv directly use os.LookupEnv, it returns variable and a bool stating if the variable is found or not
```go
envVar, found := os.LookupEnv("ritik")
```

Reply:
It assigns the value to the struct. @iamitprakash